icu4j/readme.html

```diff
@@ -879,7 +879,7 @@ <h5> Generating Data from CLDR </h5>
 <I> Note: This procedure assumes that all 3 sources are present</I>
 <ol>
     <li>Checkout or download CLDR version 'release-36'</li>
```

Comment:
CLDR version 37 here.

Reply:
Thanks - Updated this, and dropped another reference to CLDR 36